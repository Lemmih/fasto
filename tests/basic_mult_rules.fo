fun int main() =
  let x = write(5+10*4) in
  let y = write(1*2+3*4*5*6) in
  write(10*4+5)
