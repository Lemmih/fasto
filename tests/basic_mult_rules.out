45 362 45 
